### PR TITLE
added threshold option to multichainBalance

### PR DESCRIPTION
**Pull request by lzalvin**
- For multichain, users can add a `quorumThreshold` to modify the quorum of the total balance returned from the contract calls `e.g. total * 0.05`
image

### CODE REVIEW

**Comment by mktcode** [comment]
Looks good. I'm just not sure about the term `quorumThreshold`. Something like `quorumModifier` feels better to me.

**Comment by lzalvin** [comment]
@mktcode updated to `quorumModifier`

**Comment by samuveth** [comment]
Will merge once you approve @mktcode

**Review by mktcode — APPROVED** [review]

**Pull request merged**
